src/util/numeric.h

```diff
@@ -243,6 +243,11 @@ inline s32 myround(f32 f)
 	return (s32)(f < 0.f ? (f - 0.5f) : (f + 0.5f));
 }
 
+inline constexpr f32 sqr(f32 f)
```

Comment:
the name of this is unfortunate, looks just like a misspelling of `sqrt`
what's wrong with `square()`?

Reply:
It’s called `sqr` in Pascal (the only language I know with such function in the standard library). That was the only reason to choose that name.

P.S. GH email handling becomes worse every day IMO.

Reply:
'sqr' is fine for me, at least it's short.